traitsui/qt4/check_list_editor.py

```diff
@@ -199,15 +199,23 @@ def rebuild_editor(self):
         n = len(labels)
         cols = self.factory.cols
         rows = (n + cols - 1) // cols
+        # incr will keep track of how to increment index so that as we traverse
+        # the grid in row major order, the elements are added to appear in
+        # column major order
         incr = [n // cols] * cols
         rem = n % cols
         for i in range(cols):
             incr[i] += rem > i
         incr[-1] = -sum(incr[:-1]) + 1
```

Comment:
It is very strange that this line isn't `incr[-1] = -(reduce(lambda x, y: x + y, incr[:-1], 0) - 1)` like it is in all the other files which do the same thing.